feat: add documentation for go sdk (#501)
matter-labs/zksync-web-era-docs: docs/api/go/README.md

# Go SDK

> While most of the existing SDKs should work out of the box, deploying smart contracts or using unique zkSync features, like paying fees in other tokens, requires providing additional fields to those that Ethereum transactions have by default.

Reviewer: ```suggestion
While most of the existing SDKs should work out of the box, deploying smart contracts or using unique zkSync features, such as paying fees in other tokens, requires providing additional fields to those that Ethereum transactions have by default.
```

> To provide easy access to all the features of zkSync Era, the `zksync2` Python SDK was created.

Reviewer: ```suggestion
To provide easy access to all the features of zkSync Era, the `zksync2` Go SDK was created.
```

## Table of contents

- [Getting started](./getting-started.md)
- [Providers](./providers.md)
- [Accounts: Overview](./accounts.md)
- [Accounts: L1->L2 transactions](./accounts-l1-l2.md)

> - [Contracts Interface](./contract-interface.md)

Reviewer: ```suggestion
- [Contract interface](./contract-interface.md)
```
